images/grafana-rollout-operator/README.md

```diff
@@ -0,0 +1,51 @@
+<!--monopod:start-->
+# grafana-rollout-operator
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/grafana-rollout-operator` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/grafana-rollout-operator/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+Kubernetes Rollout Operator
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/grafana-rollout-operator:latest
+```
+<!--getting:end-->
+
+<!--body:start-->
+
+How to install via Helm:
+
+```bash
+helm repo add grafana https://grafana.github.io/helm-charts
+kubectl create ns grafana-rollout-operator
+helm install grafana-rollout-operator garafana/rollout-operator
```

Comment:
I think this needs to use the cgr.dev/chainguard/grafana-rollout-operator image, otherwise it is just the default from the helm chart.